packages/cli/src/__tests__/lib/commands/history-util.test.ts

```diff
@@ -201,4 +211,147 @@ describe('devices-util', () => {
 			expect(stdOutSpy).toHaveBeenCalledTimes(2)
 		})
 	})
+
+	describe('calculateHistoryRequestLimit', () => {
+		it('returns limit if less than or equal to maxHistoryItemsPerRequest', () => {
```

Comment:
This should be `equal to maxItemsPerRequest` rather than `maxHistoryItemsPerRequest`, shouldn't it?  Should the test itself use the values of 300, 301, and 500 literally, or key it off of the value of `maxItemsPerRequest`?

Reply:
Oops, fixed the incorrect name in the description.

I slightly prefer not keying off things in the code under test. In general, I like to hard-code strings and numbers in unit tests to make them break more easily when things change as well as to make them easier/more simple to read.